sei_dbt/README.md

```diff
@@ -46,14 +42,45 @@ Follow these steps to set up and configure the project:
    git clone <your-repository-url>
    cd <project-folder>
    ```
+   
+2. **Install the required Python packages**
+
+After you clone the repo, make sure there is `requirment.txt` file.
+
+In this project, we recommend using a **Conda environment** to manage dependencies and keep your Python environments clean. 
+
+To get started, create a new Conda environment from the command line with the following command: `conda create -n <env_name> python=<version>`. 
+
+Once the environment is created, activate it with `conda activate <env_name>`. 
+
 
-2. **Initiate a new projec:**
+You can then install all necessary dependencies within this environment using
 
-If you start a new project, 
+
+```bash
```

Comment:
It is not best practice to mix conda and pip together - we can address this later on however.

Reply:
Right. Switch to conda install.